src/lib/y2ntp_client/client/finish.rb

```diff
@@ -4,13 +4,16 @@
 require "installation/finish_client"
 
 Yast.import "NtpClient"
+Yast.import "Pkg"
 
 module Y2NtpClient
   module Client
     # Client to write ntp configuration at the end of installation
     class Finish < Installation::FinishClient
       include Yast::I18n
 
+      REQUIRED_PACKAGE ||= "chrony".freeze
```

Comment:
same here. And also duplicite. What about adding this constant to NtpClient? It is there as variable, but it can be constant in one place.